index.html

```diff
@@ -272,134 +315,188 @@ <h2>Glossary</h2>
+	
+<p><dfn class="lint-ignore export">Othographic syllable</dfn> This term is not clearly defined in the Unicode Standard. For their materials, the i18n group defines it to mean a typographic unit that includes more than one [=grapheme cluster=]. This is commonly the case for Brahmi-derived scripts, such as Devanagari, or Balinese, when they form conjuncts or stacks.</p>
 
-<p><dfn id="def_mojibake" class="lint-ignore export">Mojibake</dfn> (<span lang="ja">文字化け</span>). Garbled or incorrectly rendered or processed characters, generally caused by using the wrong <a>character encoding</a> to interpret the bytes in a string or file. The word is Japanese in origin and is pronounced <q>/mo.d&#x0361;&#x0292;i.ba.ke/</q>. For example, the word <q><span lang="ja">&#x6587;&#x5B57;&#x5316;&#x3051;</span></q> encoded as UTF-8 might be displayed as <q>&#x00e6;&#x0096;&#x0087;&#x00e5;&#x00ad;&#x0097;&#x00e5;&#x008c;&#x0096;&#x00e3;&#x0091;</q> if viewed in an application that thinks (incorrectly) that the character encoding is <code>ISO-8859-1</code>.</p>
 
 
 
 <div class="letter_anchor" id="n">N</div>
 	
-<p><dfn data-lt="natural language|language|Natural Language" id="def_natural_language"  class="lint-ignore export">Natural Language</dfn> (sometimes just <em>language</em>) refers to the spoken, written, or signed communications used by human beings. See also <a class="termref" href="#def_localizable_text">localizable text</a> and <a class="termref" href="#def_syntactic_content">syntactic content</a>.</p>
+<p><dfn class="lint-ignore export">Natural Language</dfn> (sometimes just <em>language</em>). Refers to the spoken, written, or signed communications used by human beings. See also [=localizable text=] and [=syntactic content=].</p>
+
+<p><dfn class="lint-ignore export">Non-linguistic Field</dfn>. Any element of a data structure not intended for the storage or interchange of natural language textual data. This includes non-string data types, such as booleans, numbers, dates, and so forth. It also includes strings, such as program or protocol internal identifiers. This document uses the term <em>field</em> as a short hand for this concept.</p>
 
-<p><dfn data-lt="non-linguistic field|field|non-linguistic fields|fields|Non-linguistic Field" id="def_non_linguistic_field" class="lint-ignore export">Non-linguistic Field</dfn>. Any element of a data structure not intended for the storage or interchange of natural language textual data. This includes non-string data types, such as booleans, numbers, dates, and so forth. It also includes strings, such as program or protocol internal identifiers. This document uses the term <em>field</em> as a short hand for this concept.</p>
+<p><dfn class="lint-ignore export" data-lt="normalisation|Unicode normalization">Normalization</dfn>. The process of removing alternate representations of equivalent sequences from textual data, to convert the data into a form that can be binary-compared for equivalence.</p>
```

Comment:
I might add a note to this, particularly since the `data-lt` includes "Unicode normalization" into this term.

```suggestion
<p><dfn class="lint-ignore export" data-lt="normalisation|Unicode normalization">Normalization</dfn>. The process of removing alternate representations of equivalent sequences from textual data, to convert the data into a form that can be binary-compared for equivalence. In internationalization contexts, this usually refers to the application to a string of one of the Unicode normalization forms, such as [=NFC=]. For more info, see [[CHARMOD-NORM]] or [[UAX15]]</p>
```

Reply:
Done